.php_cs

```diff
@@ -38,46 +38,47 @@ $config->setRules(
     [
         '@PSR2' => true,
         'align_multiline_comment' => ['comment_type' => 'all_multiline'],
+        'array_indentation' => true,
         'array_syntax' => ['syntax' => 'short'],
         'blank_line_before_statement' => [
             'statements' => ['return', 'continue', 'break', 'declare', 'throw', 'try'],
         ],
-        'cast_spaces' => ['space' => 'none'],
-        'combine_consecutive_issets' => true,
-        'combine_consecutive_unsets' => true,
+        'binary_operator_spaces' => true,
+        'cast_spaces' => ['space' => 'single'],
+        'class_attributes_separation' => ['elements' => ['method']],
         'compact_nullable_typehint' => true,
         'concat_space' => ['spacing' => 'one'],
-        'dir_constant' => true,
-        'function_to_constant' => true,
+        'declare_equal_normalize' => true,
+        'explicit_indirect_variable' => true,
+        'explicit_string_variable' => true,
+        'fully_qualified_strict_types' => true,
         'function_typehint_space' => true,
         'heredoc_to_nowdoc' => true,
         'include' => true,
         'linebreak_after_opening_tag' => true,
         'list_syntax' => ['syntax' => 'short'],
         'lowercase_cast' => true,
         'magic_constant_casing' => true,
-        'mb_str_functions' => true,
+        'method_chaining_indentation' => true,
         'method_separation' => true,
-        'modernize_types_casting' => true,
+        'multiline_whitespace_before_semicolons' => true,
         'native_function_casing' => true,
-        'native_function_invocation' => true,
```

Comment:
👎

Reply:
As showed out in #1, it's a pointless rule - all benchmarks show that at best you will save few μs on thousands of invocations, so I see absolutely no reason to force it.

Reply:
Setting aside the performance stuff here (which is debatable) - the reasoning behind this, again - is to benefit the user when copying and pasting / refactoring..  For the most part - we shouldn't have overriden functions in namesapces - however, some places might have them... Now - what happens when we accidentally have a namespaced function to do with passwords, or similar?  I'd rather make sure that these calls are going where they're intended.